src/classes/EnemyBase.js

```diff
@@ -0,0 +1,22 @@
+import { AiGameObject } from "./AiGameObject.js";
+import { Layer } from "./managers/Layer.js";
+import { NewMediaManager } from "./managers/NewMediaManager.js";
+
+export class EnemyBase extends AiGameObject {
+    constructor(xLocation, yLocation, playerShip) {
+        /**
+         * The width, height, angle (which doesn't really apply), sprite, velocityX, velocityY, collisionRadius, and mass are always the same for an EnemyBase.
+         * While the xLocation and yLocation values are also always the same when a new EnemyBase is created, leaving that as a value to pass in for now.
```

Comment:
This is technically not true anymore as the enemy base location is not fixed but depends upon player base location

- [ ] Update comment about enemy base location